packages/contracts/contracts/voting/addresslist/AddresslistVoting.sol

```diff
@@ -45,25 +46,13 @@ contract AddresslistVoting is MajorityVotingBase {
     /// @notice Initializes the component.
     /// @dev This method is required to support [ERC-1822](https://eips.ethereum.org/EIPS/eip-1822).
     /// @param _dao The IDAO interface of the associated DAO.
-    /// @param _supportThreshold The support threshold in percent.
-    /// @param _minParticipation The minimum participation ratio in percent.
-    /// @param _minDuration The minimal duration of a vote in seconds.
-    /// @param _members The initial member addresses to be listed.
+    /// @param _pluginSettings The plugin settings.
     function initialize(
         IDAO _dao,
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower,
+        PluginSettings calldata _pluginSettings,
```

Comment:
Same, shouln't plugins settings be specific to AddressList?  What works for AddressList won't work for ERC20 of ERC721

Reply:
They are designed to work for all MajorityVoting classes.

`minProposerVotingPower` is needed for `TokenVoting` and `AddresslistVoting`.

Keep in mind that people can and do often own multiple NFTs and that the voting power of a voter is not always one.
E.g. in Nouns DAO, people with more Nouns NFTs have more privileges than those with only one.

For `AddresslistVoting` (`votingPower` can be 0 or 1 if you are listed), we use it to allow non-listed people to create a proposal. 
Proposal creation gas costs might be spam protection enough for some DAOs.

Generally and for the future, I think a `PermissionCondition` makes more sense here and we should think about a `Plugin` inheriting from `PermissionCondition` that allows you to specify granular conditions for proposal creation for different plugins installed in your DAO.

Reply:
But, how can totally different governance plugins be sharing the exact same struct for being configured?
In OVOTE we may have censuses which are composed of two ERC20 voting + three ERC721.... How you even determine a balance? 

If there has to be a common interface for everything, I would use a general `bytes memory data` and let every governance plugin decode whatever info they need.

I don't even understand how it's possible to configure AddressList with an `voters: address[] list` while we are having `token: tokenAddress` in TokenVoting below.... all in the same struct  🤔

Reply:
> But, how can totally different governance plugins be sharing the exact same struct for being configured?

> If there has to be a common interface for everything

I think there is a misunderstanding. `IMajorityVoting` is not the interface for every voting plugin. It is the interface specifically for MajorityVoting classes (`TokenVoting` and `AddresslistVoting`).
The original task was about fixing the execution logic for majority voting and this is why we introduced the parameters to specify the `voteMode` as well as `minProposerVotingPower`. I don't think that every vote will use these parameters.

I think, however, that we need some form of standardized events, regarding proposals, activity, and membership across governance plugins to display the governance activity in the DAO.

Reply:
This was resolved in a call. We both agree that settings should be specific for each plugin (and that TokenVoting and AddresslistVoting share commonalities)